services/site/src/components/challengePage/Card.tsx

```diff
@@ -83,8 +83,9 @@ const Card: React.FunctionComponent<CardProps> = ({
       )}
```

Comment:
I don't know when this happened, but the classes that are currently on the article element need to go on the li element that is wrapping the article. otherwise, the cards are not responsive. You can compare it with the live site to see what I mean.